clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -322,6 +323,46 @@ static bool actionRequiresCodeGen(BackendAction Action) {
          Action != Backend_EmitLL;
 }
 
+static std::string flattenClangCommandLine(ArrayRef<std::string> Args,
+                                      StringRef MainFilename) {
+  if (Args.empty())
+  {
```

Comment:
The braces are unnecessary here, but more importantly, they should be coddled.

Reply:
Thx. I removed the braces for one line ifs completely. I had overlooked that in the coding guidelines.